examples/xgboost/cyclic/README.md

```diff
@@ -0,0 +1,26 @@
+# Cyclic Training of XGBoost
+
+"Cyclic XGBoost" is one way of performing federated tree boosting with multiple sites: at each round of tree boosting, instead of relying on the whole data statistics collected from all clients, the boosting relies on only 1 client's local data. The resulting tree sequence is then forwarded to the next client for next round's boosting. Such training scheme have been proposed in literatures [1] [2].
+
+### Run the experiment
+Log into the Admin client by entering `admin` for both the username and password.
+Then, use these Admin commands to run the experiment:
```

Comment:
This is only for "POC" mode.

Other examples are specifying:

```
### 2. Set up your FL workspace

Follow the [Quickstart](https://nvflare.readthedocs.io/en/main/quickstart.html) instructions to set up your POC ("proof of concept") workspace.

### 3. Run the experiment

Log into the Admin client by entering `admin` for both the username and password.
Then, use these Admin commands to run the experiment:


```

Reply:
Changed the readme to POC, similar to prostate/brats examples